src/theory/theory_engine.h

```diff
@@ -349,20 +349,6 @@ class TheoryEngine : protected EnvObj
    */
   Node getModelValue(TNode var);
 
-  /**
-   * Get relevant assertions. This returns a set of assertions that are
-   * currently asserted to this TheoryEngine that propositionally entail the
-   * (preprocessed) input formula and all theory lemmas that have been marked
-   * NEEDS_JUSTIFY. For more details on this, see relevance_manager.h.
-   *
-   * This method updates success to false if the set of relevant assertions
-   * is not available. This may occur if we are not in SAT mode, if the
-   * relevance manager is disabled (see option::relevanceFilter) or if the
-   * relevance manager failed to compute relevant assertions due to an internal
-   * error.
-   */
-  std::unordered_set<TNode> getRelevantAssertions(bool& success);
```

Comment:
Can you keep this? I had a request for this functionality at some point but haven't connected it to the API yet.

Reply:
Sounds good, I've added it back. I've changed it slightly (to not depend on `d_inSatMode` and moved it to the same relative place as its declaration in the header file.